docs-kits/kits/Digital Twin Kit/Software Development View/page_interaction-patterns.md

```diff
@@ -0,0 +1,262 @@
+
+The Catena-X standard CX-0002 (Digital Twins in Catena-X) defines a subset of the AAS-standard that is relevant for 
+Catena-X. However, it is concerned only with the network-facing APIs which in some cases may have to be augmented to 
+cover use-case requirements.
+
+While the assumptions are explicitly listed, all patterns assume that both partners involved are properly onboarded to
+the Catena-X dataspace and thus have their connectors registered with the EDC Discovery Service like so:
+
+```mermaid
+sequenceDiagram
+    participant PAdmin as Provider Setup Admin
+    participant EDCD as EDC Discovery
+    participant CAdmin as Consumer Setup Admin
+autonumber
+    critical: setup of registration
+        PAdmin ->> EDCD: POST /api/administration/connectors <br/> with link between BPN and EDC endpoint
+        CAdmin ->> EDCD: POST /api/administration/connectors <br/> with link between BPN and EDC endpoint
+    end
+```
+This happens usually when onboarding a company to the network.
+
+### 1. Fetching a supplier's Twin
+
+| Scenario                                                                    | Participants                                           | Assumptions                                                                 | Links to Use-Cases                                                                               |
+|-----------------------------------------------------------------------------|--------------------------------------------------------|-----------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
+| The supplier has data required by the OEM. For instance a Product Passport. | - Data Provider (Supplier) <br/> - Data Consumer (OEM) | 1. Consumer knows Supplier's BPN <br/> 2. Consumer knows an id of the asset | - Industry Core (coming soon) <br/> - [Product Carbon Footprint Kit](../../PCF%20Exchange%20Kit) |
+
+
+The most common integration pattern with Digital Twins aims to offer asset-related data in expectable ways along the
+supply chain. It defines conventions that allow locating data in the network. This most basic scenario assumes that the 
+supplier offers data about an asset that was sold to an OEM further up the chain. Supplier and OEM have agreed on 
+contractual terms on which to share data about said asset. Possible examples may include Bills of Material, carbon 
+footprints, technical specifications or regulatory documents. An OEM - or really any interested and privileged third 
+party - can assume that a part's supplier offers a Digital Twin if the data-exchange is based on the Digital Twin Kit 
+(and by proxy CX-0002). The data provider deploys at least:
+- Digital Twin Registry (DTR)
+  - registered in a DSP-conformant connector offering the DTR to the Dataspace.
+  - registers Digital Twins (AssetAdministrationAAS-descriptors - or AAS descriptor - in AAS vocabulary) with assetIds 
```

Comment:
```suggestion
  - registers Digital Twins (AssetAdministrationAAS-descriptors - or AAS descriptor - in AAS vocabulary) with assetI IDs
```
```suggestion
  - registers Digital Twins (AssetAdministrationAAS-descriptors - or AAS descriptor - in AAS vocabulary) with asset IDs
```

Reply:
There is not "assetId" attribute, there is the globalAssetId and there are specificAssetIds.

Reply:
part 2 of the AAS spec, p. 113, bullet point 13 introduces the term as the union of specficAssetIds and globalAssetIds.